source/h5vol/CMakeLists.txt

```diff
@@ -9,6 +9,21 @@ if(CMAKE_C_COMPILER_ID MATCHES "^(GNU)$")
   string(APPEND CMAKE_C_FLAGS " -Wno-format-zero-length")
 endif()
 
+include(CheckIncludeFiles)
+check_include_files(unistd.h HAVE_UNISTD_H)
+check_include_files(windows.h HAVE_WINDOWS_H)
+
+message("- HAVE_UNISTD?  ${HAVE_UNISTD_H} ")
```

Comment:
I'd probably kill these "message" lines.  The check_include_files will include verbosity about checking and whether or not it was found.  No reason to add yet another line to the two lines of output that are already generated by that...

Reply:
Okay!